Pages/Sessions.razor

```diff
@@ -28,14 +28,27 @@
                         <thead>
                             <tr>
                                 <th>Id</th>
+                                <th>Page</th>
+                                <th>Authors</th>
+                                <th>Speaker</th>
                             </tr>
                         </thead>
                         <tbody>
                             @foreach (var lecture in session.Lectures)
                             {
                                 <tr>
                                     <td>@lecture.Id</td>
-                                </tr>
+                                    @if (@lecture is not null) {
+                                        <td>@lecture.Paper.Name</td>
+                                        <td>@GetAuthorNameList(lecture.Paper)</td>
+                                        <td>@lecture.Speaker.DisplayName</td>
```

Comment:
Z kolei jeśli `lecture.Paper` lub `lecture.Speaker` są `null`  to raczej nie chcemy wykrzaczyć całego frontendu. Lepiej byłoby `lecture.Paper?.Name` i `lecture.Speaker?.DisplayName`. Nie ignorujmy warningów.